docs/quickstart.rst

```diff
@@ -295,7 +295,7 @@ Python shell. See :ref:`context-locals`.
 
     @app.route('/user/<username>')
     def profile(username):
-        return '{}\'s profile'.format(username)
+        return '{}\'s profile'.format(escape(username))
```

Comment:
Other examples also import `escape` from flask, I don't see why this shouldn't also.

Reply:
Yes, my bad, missed this when reading through since the other examples I'm editing here are fragments without imports, but this is a whole file. Fixed!